specifications/api/README.md

```diff
@@ -13,6 +13,11 @@ $ python -m SimpleHTTPServer
 Serving HTTP on 0.0.0.0 port 8000 ...
 ```
 
+```
+$ python3 -m http.server
```

Comment:
Just delete the python2 version. All distros ship python3.

Reply:
Piotr say he has Python2 :upside_down_face: I can also in my Arch Linux set any one as default. Maybe it is worth leaving both versions